engine/src/main/resources/assets/i18n/menu_id.lang

```diff
@@ -0,0 +1,255 @@
+{
+    "activate-module": "Aktifkan",
+    "add-server": "Tambah baru",
+    "add-server-tip": "Anda bisa menekan ENTER untuk mengcopy nama ke kotak alamat",
+    "advanced-usage": "Penggunaan tingkat lanjut!",
+    "animate-grass": "Animasikan Rumput",
+    "animate-water": "Animasikan Air",
+    "audio-settings": "Suara",
+    "audio-settings-title": "Pengaturan Suara",
+    "back": "Kembali",
+    "behavior-editor-assign": "masukan",
+    "behavior-editor-copy": "copy",
+    "behavior-editor-layout": "rancangan",
+    "behavior-editor-new": "baru",
+    "behavior-editor-paste": "paste",
```

Comment:
But I'll change it to your suggestion since it's valid and some popular application also uses it.